Readme.md

```diff
@@ -123,7 +123,7 @@ clientId | String | Amazon Client ID
 button | Object/String | [button options](#button-options). If String, sets button.id
 wallet | Object/String | [wallet options](#wallet-options). If String, sets wallet.id
 [addressBook] | Object/String | [addressBook options](#addressbook-options). If excluded, customers will interact solely with the wallet and consent widgets. If String, sets addressBook.id
-consent | Object/String | [conset options](#consent-options). If String, sets consent.id
+[consent] | Object/String | [conset options](#consent-options). If excluded, customers will not need to give a consent. If String, sets consent.id
```

Comment:
**question**:  Did we go the opposite direction from what I was expecting?

Just to confirm:  It's my expectation that the consent widget defaults or "falls back" to presenting the checkbox for the customer to consent to terms during checkout.  That is, today's historical behavior is preserved unless it is specifically overridden to be hidden.  

The way this reads, it feels like the default behavior will be to hide that consent form/checkbox from the customer unless `true` is passed via the `[consent]` option.